README.md

```diff
@@ -98,9 +98,11 @@ Table of Contents
 
 ## Learn More
 Read the [documentation](https://glab.readthedocs.io/) for more information on this tool.
+### Documentation
+Get the new documentation on [`https://glab-cli.io`](https://glab-cli.io) and contribute to the new documentation on [https://github.com/braswelljr/glab-docs](https://github.com/braswelljr/glab-docs).
```

Comment:
Remove the documentation header. 

I'm thinking maybe we could move the docs from the https://github.com/braswelljr/glab-docs repo to the /docs folder in this repo.